lib/Runtime/PlatformAgnostic/Intl.h

```diff
@@ -5,6 +5,48 @@
 
 #pragma once
 
+namespace PlatformAgnostic
+{
+namespace Intl
+{
+    // NOTE(jahorto): Keep these enums in sync with those by the same name in Intl.js
+    enum class NumberFormatStyle
+    {
+        DEFAULT = 0, // "decimal" is the default
+        DECIMAL = 0, // Intl.NumberFormat(locale, { style: "decimal" }); // aka in our code as "number"
+        PERCENT = 1, // Intl.NumberFormat(locale, { style: "percent" });
+        CURRENCY = 2, // Intl.NumberFormat(locale, { style: "currency", ... });
+        MAX
+    };
+    enum class NumberFormatCurrencyDisplay
+    {
+        DEFAULT = 0, // "symbol" is the default
+        SYMBOL = 0, // Intl.NumberFormat(locale, { style: "currency", currencyDisplay: "symbol" }); // e.g. "$" or "US$" depeding on locale
+        CODE = 1, // Intl.NumberFormat(locale, { style: "currency", currencyDisplay: "code" }); // e.g. "USD"
+        NAME = 2, // Intl.NumberFormat(locale, { style: "currency", currencyDisplay: "name" }); // e.g. "US dollar"
+        MAX
+    };
+
+    enum class CollatorSensitivity
+    {
+        DEFAULT = 0,
+        BASE = 1,
+        ACCENT = 2,
+        CASE = 3,
+        VARIANT = 4,
+        MAX
+    };
+    enum class CollatorCaseFirst
+    {
+        DEFAULT = 0,
+        UPPER = 1,
+        LOWER = 2,
+        NONE = 3, // this is "FALSE" on the JS-side, but we cant use FALSE here because it confuses the preprocessor
```

Comment:
You could get around this FALSE confusion by using long-names for your enum values. Something like:

```c++
enum class CollatorCaseFirst : uchar
{
    CollatorCaseFirst_Default = 0,
    CollatorCaseFirst_Upper = 1,
    CollatorCaseFirst_Lower = 2,
    CollatorCaseFirst_False = 3,
    CollatorCaseFirst_Max
}
```

This can look pretty verbose, though, but it does help readability. We don't really have a standard guide around enum naming conventions so this is up to you. 👍

Reply:
If I went with that route, then I wouldn't use an enum class at all. If I took your suggestion to pascal-case these enum values, then False wouldnt confuse the preprocessor in the same way that FALSE and false would.

Reply:
enum class members require the namespace prefix when used (or otherwise requires e.g. `using CollatorCaseFirst`).

I think you should probably use `False`. Unfortunate about the possible name collisions but it comes from the spec.